packages/core/src/common/_variables.scss

```diff
@@ -141,7 +141,7 @@ $pt-dark-input-intent-box-shadow-colors: (
   "primary": $blue4,
   "success": $green4,
   "warning": $orange4,
-  "danger" : $red4,
```

Comment:
this was causing the regex to be tripped up... I've removed the extra whitespace, but also added code to handle it if necessary